Tools/CMake/Sign.cmake.in

```diff
@@ -0,0 +1,55 @@
+# This file is generated from `Sing.cmake.in`
```

Comment:
Sign.cmake.in?

la laaaaa la la

Reply:
You have no idea how many time I misspelled that in the body of code, and wonder why things doesn't work! 😫